pkgs/development/python-modules/tensorboardx/default.nix

```diff
@@ -19,6 +19,12 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ numpy protobuf six ];
 
+  # apparently torch API changed a bit at 1.6
```

Comment:
Thanks for fixing this!

Could you file an upstream issue (if there isn't one already) and then add the URL of the issue here? It will make it easier in the future to track the status of PyTorch 1.6.0 support.

Reply:
Current upstream version is 2.1, somehow it doesn't have this particular test failure but has a plenty of others. Official page lists `python-3.6` and `pytorch-1.5` as being supported and tested, so maybe we're expecting to much :)

Personally, I found even `tensorboardX-1.6` more than enough for my needs, so I propose to wait for upstream to catch up with recent dependencies versions.

Maybe some important functionality is missing for someone? When it's another question.

Reply:
Sorry, I misinterpreted

> Personally, I found even `tensorboardX-1.6` more than enough for my needs, so I propose to wait for upstream to catch up with recent dependencies versions.

as that you you wanted to wait until upstream patches for PyTorch 1.6.